styles/main.scss

```diff
@@ -579,10 +578,6 @@ header {
   overflow: hidden;
 }
 
-.posts-list-block:hover .post-img:after {
-  height: 100%;
-}
-
```

Comment:
![animation](https://user-images.githubusercontent.com/46556479/202861501-c3a488c9-8d25-45bb-8986-9ec3dcf68fe2.gif)
hover effect is broken